src/__test__/index.test.ts

```diff
@@ -62,3 +75,61 @@ describe('exports', () => {
 		expect(ignoreVisit.mock.calls).toHaveLength(1);
 	});
 });
+
+describe('Plugin module', () => {
```

Comment:
Please feel free to move these around into separate files if you wish. This file already starts to grow quite a lot. 
I think these could be in `plugins.test.ts` next to the `plugins.ts` file, for example.

Reply:
Good point, will address this in a separate PR.